### ISSUE TITLE
Cascading LOV child throws Not all variables bound

### ISSUE THREAD

**Issue by Cristian-I** [bostrowski/Pretius-APEX-Enhanced-Lov-Item#14]
Hey @bostrowski 

Thank you for this wonderful plugin. I'm getting the error "Not all variables bound" in an cascading lov item after i submit the page. 
You can test this here: https://apex.oracle.com/pls/apex/f?p=19101 
Steps:
- Select a department    (`P1_DEPARTMENT`)
- Select an emp   (this has Cascading LOV Parent:`P1_DEPARTMENT` and sql query `select empno r, ENAME d from emp where deptno=:P1_DEPARTMENT`)
- Click Save (this will just submit the page without any process)
- And when the pages is rendered again the Emp will just display the retrieve value of the emp as the pkg will throw "Not all variables bound" when trying to open the cursor at line 866 because the v_query contains the LOV parent variable (`:P01_DEPARTMENT` in this example).

Exported app: [f19101.zip](https://github.com/bostrowski/Pretius-APEX-Enhanced-Lov-Item/files/3472457/f19101.zip)

Let me know if i can provide any more info. 

Thank you

**Comment by Cristian-I** [comment]
I found the solution. Using `V` / `NV` Functions instead of bind variables.
I changed the query to `select empno r, ENAME d from emp where deptno=v('P1_DEPARTMENT')`.  

All is good now but i think you have to specify this somewhere in the readme.

PS. Maybe this is also the solution for #12

**Comment by bostrowski** [comment]
@Cristian-I 
Thank you for your detailed feedback. I will investigate it closely and get back to this thread.

**Comment by bostrowski** [comment]
@Cristian-I 
You should consider it as bug. The proper fix is on the way, but firstly I have to finish other fixes for `v1.0.5`.

If you want to fix it yourself you should update the plugin package by replacing line 866 which is
```sql
open v_ref_cursor for v_query;
```

with 

```sql
v_ref_cursor := getBindedRefCursor(v_query);
```

thanks to this you won't need to use `V` function.

Ps.
your demo app was helpful, thank you!
